crm_claim_delivery/models/product_template.py

```diff
@@ -33,7 +33,7 @@ def _compute_claim_ids(self):
     )
     total_claims = fields.Integer(
         compute='_get_total_claims',
-        store=False
+        store=True
```

Comment:
Put a , (comma) after store=True and we are ready to go.

Otherwise you would need to change this line again, when putting in another keyword